src/HttpServer.ts

```diff
@@ -134,6 +202,37 @@ export class HttpServer {
         }
     }
 
+    /**
+     * @openapi
+     * /tokens:
+     *   get:
+     *     summary: It retrieves the accepted tokens.
+     *     description: "It retrieves the accepted tokens of the specified verifier if any, otherwise, it retrieves the accepted tokens of all the verifiers in the format { <verifier_address>: [accepted_token_address_1, accepted_token_address_2]}"
+     *     parameters:
+     *       - in: path
+     *         name: verifier
+     *         required: false
+     *         description: The address of the verifier to use to retrieve the accepted tokens.
```

Comment:
i'm not sure if this is a swagger or server code error, but providing a value (even an invalid value) seems to have no effect when testing out the result of this call in the docs page.

have you experienced the same?

Reply:
Good catch! It was the documentation that was configured wrongly, thank you!